dts/arm/nxp/nxp_kw2xd.dtsi

```diff
@@ -57,8 +57,9 @@
 		};
 
 		flash-controller@4001f000 {
-			compatible = "nxp,k64f-flash-controller";
-			reg = <0x4001f000 0x27c>;
+			compatible = "nxp,kinetis-ftfl";
+			label = "FLASH_CTRL";
+			reg = <0x4001f000 0x240>;
```

Comment:
Compatible looks good, let's also update the base address to the FTFL_BASE=0x40020000 rather than the FMC_BASE. You already made the same change for k6x

Reply:
Agreed.